types/wlr_xdg_output_v1.c

```diff
@@ -132,6 +132,10 @@ static void output_manager_handle_get_xdg_output(struct wl_client *client,
 	}
 
 	output_send_details(xdg_output, xdg_output_resource);
+
+	if (version => OUTPUT_DONE_DEPRECATED_SINCE_VERSION) {
```

Comment:
Hmm, I have no idea how we missed that.

Let's schedule a `wl_output.done` unconditionally, just like in `output_update`.